cmd/make.go

```diff
@@ -199,11 +259,33 @@ func runTestcases() {
 
 func clean() {
 	gopBinPath := detectGopBinPath()
-	if checkPathExist(gopBinPath, true) {
-		if err := os.RemoveAll(gopBinPath); err != nil {
-			println(err.Error())
+	goBinPath := detectGoBinPath()
+
+	if !checkPathExist(gopBinPath, true) {
+		return
+	}
+
+	gopBinFiles, err := os.ReadDir(gopBinPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// Clean links
+	for _, file := range gopBinFiles {
+		if !file.IsDir() {
+			targetLink := filepath.Join(goBinPath, file.Name())
+			if checkPathExist(targetLink, false) {
+				if err := os.Remove(targetLink); err != nil {
+					log.Fatalln(err)
+				}
+			}
 		}
 	}
+
+	// Clean build binary files
+	if err := os.RemoveAll(gopBinPath); err != nil {
```

Comment:
maybe don't remove old results (so that don't break links in goBinPath)?

Reply:
Now, only clean when uninstall.

Reply:
Now, only clean when uninstall.